service-base-utils-bld-os/run_configuration/selectedProjectMavenCleanInstall.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?><launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
```

Comment:
This looks like an Eclipse thing, is it worth including?

Reply:
These are handy utilities in eclipse, I would like to keep them and I think the common services team wants them as well.